src/EnergyPlus/PlantComponentTemperatureSources.cc

```diff
@@ -533,38 +507,38 @@ namespace PlantComponentTemperatureSources {
 		Real64 tmpVolFlowRate; // local design volume flow rate
 		Real64 DesVolFlowRateUser( 0.0 ); // Hardsized design volume flow rate for reporting
 
-		tmpVolFlowRate = WaterSource( SourceNum ).DesVolFlowRate;
+		tmpVolFlowRate = this->DesVolFlowRate;
 
-		PltSizNum = PlantLoop( WaterSource( SourceNum ).Location.loopNum ).PlantSizNum;
+		PltSizNum = PlantLoop( this->Location.loopNum ).PlantSizNum;
```

Comment:
@Myoldmopar I meant to say . . . needs Location which is *not* being passed in by getDesignCapacities